README.md

```diff
@@ -317,7 +317,7 @@ Flags:
       --append-user-to-subdomain-separator string               The token to use for separating username and subdomain selection in a virtualhost (default "-")
       --authentication                                          Require authentication for the SSH service (default true)
       --authentication-key-request-timeout duration             Duration to wait for a response from the authentication key request (default 5s)
-  -v, --authentication-key-request-url string                   A url to validate public keys for public key authentication.
+      --authentication-key-request-url string                   A url to validate public keys for public key authentication.
```

Comment:
@rjobanp heads up, I'm removing this short flag here as viper uses `-v` by default for version output.